packages/pangea-django/.django-ci.yml

```diff
@@ -1,25 +1,36 @@
-.pangea-django-base:
+.django-base:
+  image: python:3.8.18
```

Comment:
One thing I think we can do here is use python:3.8-bullseye so it's always using the latest patch version of python with security updates. it falls in line a little more with the python version matrix in examples-ci.yml too

Reply:
Renovate can keep it updated (check it out [elsewhere](https://github.com/pangeacyber/pangea-javascript/commit/2fee0d6)), although technically we'd have to rename this to `.gitlab-ci.yml` (which is desired IMO, but I'll save that for another PR). Anyways, done.